src/main/java/datagateways/PropertyDataManager.java

```diff
@@ -0,0 +1,44 @@
+package datagateways;
+
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * The PropertyDataManager class is responsible for reading property data from a CSV file and providing it as an ArrayList of String arrays.
+ */
+public class PropertyDataManager implements DataManager{
+
+    /**
+     * Creates properties based on the information in the specified CSV file.
+     *
+     * @param fileName The name of the CSV file containing property information.
+     * @return An ArrayList of String arrays representing the data read from the CSV file.
+     */
+    public ArrayList<String[]> readData(String fileName){
+        ArrayList<String[]> lines = new ArrayList<>();
+
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))){
+            String line;
+
+            while ((line = br.readLine()) != null) {
+                String[] values = line.split(DELIMITER);
+                if (values.length < 12){
```

Comment:
This is the same as above, you can make a variable on top of the file that allows for easy change in these numbers